pykov.py

```diff
@@ -1085,7 +1085,13 @@ def walk(self, steps, start=None, stop=None):
         ['B', 'A', 'A', 'A', 'A', 'A', 'B']
         """
         if start is None:
-            start = self.steady().choose()
+            steady = self.steady()
```

Comment:
As you noted the `.steady()` method may return something even if the chain is not ergodic. I have the feeling that the returned array is not predictable. So for the time being, in order to update the documentation, I would write:

> If not indicated, then the starting state is chosen according to the steady state probability in case of an ergodic chain, otherwise the starting state is unpredictable.

Any consideration?